schemas/Interaction.json

```diff
@@ -0,0 +1,51 @@
+{
+  "type": "object",
+  "description": "Triggered when the user clicks/taps/activates a UI element.",
+  "properties": {
+    "eventName": {
+      "enum": ["Interaction"]
+    },
+    "name": {
+      "description": "The unique name of this element.",
+      "type": "string",
+      "oneOf": [
+        {"const": "ComposerSendMessage", "description": "User sent a message via the send message composer." },
```

Comment:
Maybe this one is important enough to be its own event?

Reply:
I guess it should be simply because it can have props like isEdit, isReply, isThread, etc